concourse-chrome-driver/terraform/deployments/670214072732/module-codebuild-build-container-docker-hub.tf

```diff
@@ -0,0 +1,27 @@
+module "codebuild-build-container-docker-hub-cd" {
+  source              = "github.com/alphagov/cyber-security-shared-terraform-modules//codebuild/codebuild_build_container_docker_hub"
+  stage_name          = "codepipeline-docker-image"
```

Comment:
The intention is that these should match the name of the codepipeline stage and actions 

```
  stage {
    name = "BuildDockerImages"

    action {
      name             = "BuildAndPushCdImage"
```

Reply:
Cheers. Changed to CamelCase 👍